mercadopago/px-ios
# (Added)[PXN-1782] - Adding sonic branding to the congrats screen

## What have changed?
Adding sound to the congrats animation when the pay button is `"exploding"`

## How to test:
Go to one tap flow and create a payment. After the payment successes a sound should be played while the "explosion" animation is being made.

## Prints for layout changes:
N/A

## Extras:
N/A
